samcli/lib/iac/cfn/helpers.py

```diff
@@ -0,0 +1,18 @@
+"""
```

Comment:
This file is all from Daniel's changes that I pulled

Reply:
This file has been removed in Daniel's PR, as the `get_resolved_cfn_template` function just wraps the `SamBaseProvider.get_resolved_template_dict` call and does nothing else.